README.md

```diff
@@ -84,6 +84,29 @@ If you use a URL like `s3://bucket/path`, you can have it save to an S3 bucket.
 
 Note that for s3, you'll need to specify your AWS credentials and default AWS region via `AWS_ACCESS_KEY_ID`, `AWS_SECRET_ACCESS_KEY` and `AWS_DEFAULT_REGION`
 
+#### Custom backup source file name
+There may be use-cases where you need to modify the source path of the backup file **before** it gets uploaded to the dump target. 
+An example is combining multiple compressed files into one and giving it a new name, i.e. ```db-other-files-combined.tar.gz```.
+To do that, place an executable file called ```source.sh``` in the following path:
+
+      /scripts.d/source.sh
```

Comment:
Can we add a description of what the input and output should be? It looks good for `target.sh` a few lines below, where it says, "Whatever your script returns to _stdout_ will be used as the name for the backup file".

An example also would be really helpful.